src/components/Input.tsx

```diff
@@ -1,17 +1,18 @@
+import React from 'react';
+import { Field, ErrorMessage } from 'formik';
+
 const Input = ({
```

Comment:
how can we determine if this is a required or optional field?

Reply:
For required, we create the separate validation schema and for optional, now I added the optional props in the input component.